bootstrap.go

```diff
@@ -254,7 +254,7 @@ func upgrade(s Session) {
 func upgradeToVer2(s Session) {
 	// Version 2 add two system variable for DistSQL concurrency controlling.
 	// Insert distsql related system variable.
-	distSQLVars := []string{variable.DistSQLScanConcurrencyVar, variable.DistSQLJoinConcurrencyVar}
```

Comment:
It's not used right now, we can easily add it later if needed.
We add it before hand because we used need to update bootstrap version for adding new global variable.